src/app.js

```diff
@@ -1,15 +1,20 @@
+/* eslint-disable linebreak-style */
```

Comment:
ESLint errors can get frustrating sometimes but it's pretty easy to fix. The inline comment here works, but you would be cleaner and easier to add this rule to the `.eslintrc.js` file at the bottom where the other rules have already been created like this ```'linebreak-style': 0,```

Reply:
alright fixed it